crates/credentials/src/vc.rs

```diff
@@ -127,25 +195,163 @@ impl VerifiableCredential {
         Ok(jwt)
     }
 
-    pub async fn verify(jwt: &str) -> Result<Self, CredentialError> {
+    pub async fn verify(jwt: &str) -> Result<Arc<VerifiableCredential>, CredentialError> {
```

Comment:
I'd advise against introducing an `Arc` until it's absolutely necessary, it may lead to unnecessary complexity

Reply:
ok happy to turn it off, was just sticking to the same signature it had before